webapp/components/wpt-flags.js

```diff
@@ -101,8 +108,26 @@ wpt.FlagsClass = (superClass, readOnly, useLocalStorage) => class extends superC
   static get properties() {
     const props = {};
     makeFeatureProperties(props, wpt.ClientSideFeatures, readOnly, useLocalStorage);
+    if (useLocalStorage) {
+      makeFeatureProperties(props, wpt.ClientSideWriteableFeatures, false, useLocalStorage);
+    }
     return props;
   }
+
+  ready() {
+    super.ready();
+    if (useLocalStorage) {
+      for (const feature of wpt.ClientSideWriteableFeatures) {
+        this._createMethodObserver(`valueChangedInLocalStorage(${feature}, '${feature}')`);
```

Comment:
This should be removed I think?

Reply:
Without this line, the value of `isTriageMode` won't be saved when a user refresh the page?

Reply:
Do we want that? I think we can either store every state (including the cells selected), or revert back to default (triage mode off, nothing selected) upon refresh. Remembering just the triage mode state seems not particularly useful and potentially confusing.

Reply:
Right, I can see that the triage mode should only be on when users trying to triage at the moment. Having it on by default doesn't make sense anyways (confusing on the UI)